### PR TITLE
Update span! macro to be compliant with tokio-rs/tokio#1025

### PR DESCRIPTION
**Pull request by jxs**

### CODE REVIEW

File: tokio-trace-futures/src/lib.rs [hunk 1]
```diff
@@ -4,7 +4,7 @@ extern crate tokio;
 #[cfg_attr(test, macro_use)]
 extern crate tokio_trace;
 
-use futures::{Async, Future, Poll, Sink, StartSend, Stream};
```
Comment (hawkw) [inline tokio-trace-futures/src/lib.rs]: Was this import unused before this change? Removing it is fine either way, just curious.

Comment (jxs) [reply tokio-trace-futures/src/lib.rs]: yeah, apparently. cargo gave the warning first time I run cargo build

Comment (hawkw) [reply tokio-trace-futures/src/lib.rs]: cool, thanks for fixing it!

**Comment by jxs** [comment]
sorry, forgot to run the tests, think it's fixed now

File: tokio-trace-futures/src/lib.rs (hunk 1 above)
Comment (jxs) [reply tokio-trace-futures/src/lib.rs]: funny, turns out it was being required on tests?

**Pull request merged**